### ISSUE TITLE
directory structure lost

### ISSUE THREAD

**Issue by yepyepyep4711** [TeamShinkansen/Hakchi2-CE#225]
**Describe the bug**  
probably due to a language change, the directory structure was lost. I use the custom variant, and at the next sync, every thing was in the "to be sorted" folder. If I had synced at this point, I would have lost the structure that was on the console.
**To Reproduce**  
1. make a custom directory structure and sync it
2. change languages in Hackchi
3. Add a game
4. Sync

**Expected behavior**  
That hackchi gets the directory structure from the console before a sync and then asks what changes the user want to make, as opposed to starting from scratch lie now. Also, the main list doesn't reprise the directory structure, just "own games" and in-built ones. Maybe this could be changed to display the directory structure instead? Icons could be used to differentiate between original games and the others, like in the directory structure screens.

**Comment by KMFDManic** [comment]
@yepyepyep4711 One thing I typically do when working with folders is to set them up, sort them, then "cancel", which gives me option to immediately save current folder structure.  That is then intact and ready to go next time I go to synch/export.

And, yes, it does appear some language changes have odd behavior, which is being looked into.  Which exact language were you attempting to change to?

**Comment by KMFDManic** [comment]
@yepyepyep4711 As mentioned in your duplicate issue, this is WIP fix by DanTheMan827 with 

https://github.com/TeamShinkansen/hakchi2/issues/203

Things should fare better next Hakchi Release.  For now, utilize the other suggestion I notated!  Thanks for the feedback and the suggestion:)

**Comment by DanTheMan827** [comment]
@yepyepyep4711 I was unable to replicate this issue

I did exactly like you said:

1. make a custom directory structure and sync it
2. change languages in Hakchi
3. Add a game
4. Sync

However, upon sync, the folder manager showed up exactly like expected and only my new game was in the unsorted folder.

Perhaps it's possible that the issue was inadvertently fixed by something else I changed in the 3.6 branch
